src/WebJobs.Script/ManagedDependencies/ManagedDependencyProvider.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal class ManagedDependencyProvider : IManagedDependencyProvider
+    {
+        /// <summary>
+        /// Returns the managed dependecies meta data
+        /// </summary>
+        /// <param name="hostFilePath">The configuration path for the managed dependencies</param>
+        /// <param name="managedDependencyOptions">Configurations options for managed dependencies</param>
+        /// <returns>The IEnumerable list of <see cref="Dependency"/></returns>
+        public IEnumerable<Dependency> GetDependencies(string hostFilePath,
```

Comment:
Do not use hostFilePath. Use configuration instead: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/BindingExtensionBundle/ExtensionBundleOptionsSetup.cs#L19